gator/arguments.py

```diff
@@ -171,6 +243,41 @@ def is_valid_commits(args):
         return True
     return False
 
+def is_valid_issues(args):
```

Comment:
This list is likely not correct -- there are not 6 different ways to call the check, right? These need to validate the combinations of arguments. How do you combine `--name` and `--issue`, for instance?